### ISSUE TITLE
Questions about mu calculation

### ISSUE THREAD

**Issue by lytkarinskiy** [ZhengyaoJiang/PGPortfolio#45]
Hello again!

In function  [`__pure_pc(self)`](https://github.com/ZhengyaoJiang/PGPortfolio/blob/4ccd4d21d09973fa49d747041871e852aca9eded/pgportfolio/learn/nnagent.py#L176) only initial guess for mu is used, the rest of the function is commented. Is it ok or some kind of mistake?
Also, in commented block there is part [`mu = 1-3*c+c**2`](https://github.com/ZhengyaoJiang/PGPortfolio/blob/4ccd4d21d09973fa49d747041871e852aca9eded/pgportfolio/learn/nnagent.py#L182) where you subtract `3 x commission_rate`, instead of `2 x commission_rate`, as in the paper, but in `[calculate_pv_after_commission(w1, w0, commission_rate)](https://github.com/ZhengyaoJiang/PGPortfolio/blob/4ccd4d21d09973fa49d747041871e852aca9eded/pgportfolio/tools/trade.py#L39) it seems like ok . Is it also some kind of mistake?

Thanks!

**Comment by istvanmo** [comment]
Hi there!
I have the same question. Have you got your answer yet?

**Comment by lytkarinskiy** [comment]
Hi! Not yet, seems like it's done for train speedup. During backtest it's handled as it should

**Comment by ZhengyaoJiang** [comment]
> Hi! Not yet, seems like it's done for train speedup. During backtest it's handled as it should

Sorry for the late response.
Yes, the iterative method of calculating mu will slow down the training significantly, while using `mu = 1 - tf.reduce_sum(tf.abs(w_t1[:, 1:]-w_t[:, 1:]), axis=1)*c` the difference between the approximate value and true value is only O(c^2).

**Comment by lytkarinskiy** [comment]
Thanks for the response!
